client/src/app/site/users/components/user-detail/user-detail.component.ts

```diff
@@ -0,0 +1,229 @@
+            // repo sometimes delivers undefined values
+            if (newViewUser) {
+                this.user = newViewUser;
+                // for "new" and reload
+                if (this.personalInfoForm) {
+                    this.patchValues();
+                }
+            }
+        });
+    }
+
+    /**
+     * initialize the form with default values
+     */
+    public createForm(): void {
+        this.personalInfoForm = this.formBuilder.group({
+            username: [''],
+            title: [''],
+            first_name: [''],
+            last_name: [''],
+            structure_level: [''],
+            number: [''],
+            about_me: [''],
+            groups_id: [''],
+            is_present: [true],
+            is_committee: [false],
+            email: ['', Validators.email],
+            last_email_send: [''],
+            comment: [''],
+            is_active: [true],
+            default_password: ['']
+        });
+
+        this.patchValues();
+    }
+
+    /**
+     * Loads values that require external references
+     * And allows async reading
+     */
+    public patchValues(): void {
+        this.personalInfoForm.patchValue({
+            groups_id: this.user.groupIds,
+            title: this.user.title,
+            first_name: this.user.firstName,
+            last_name: this.user.lastName
+        });
+    }
+
+    /**
+     * Makes the form editable
+     * @param editable
+     */
+    public makeFormEditable(editable: boolean): void {
+        if (this.personalInfoForm) {
+            const formGroup = [
+                this.personalInfoForm.get('username'),
+                this.personalInfoForm.get('title'),
+                this.personalInfoForm.get('first_name'),
+                this.personalInfoForm.get('last_name'),
+                this.personalInfoForm.get('email'),
+                this.personalInfoForm.get('structure_level'),
+                this.personalInfoForm.get('number'),
+                this.personalInfoForm.get('groups_id'),
+                this.personalInfoForm.get('default_password'),
+                this.personalInfoForm.get('comment'),
+                this.personalInfoForm.get('is_present'),
+                this.personalInfoForm.get('is_active'),
+                this.personalInfoForm.get('is_committee'),
+                this.personalInfoForm.get('about_me')
+            ];
+
+            if (editable) {
+                formGroup.forEach(formElement => {
+                    formElement.enable();
+                });
+            } else {
+                formGroup.forEach(formElement => {
+                    formElement.disable();
+                });
+            }
+        }
+    }
+
+    /**
+     * Handler for the generate Password button
+     * TODO: This is a lazy very unsecure pseudo password generator
+     *       Use externally or make own.
+     */
+    public generatePassword(): void {
```

Comment:
Please use letters. see this (very similar) implementation https://github.com/OpenSlides/OpenSlides/blob/master/openslides/users/static/js/users/site.js#L188